Add client example (#11)
Azure-Samples/quote-of-the-day-javascript: client-app/README.md

# Quote of the Day

A react application showing how to load a **variant feature flag**  from **Azure App Configuration** via **Azure Front Door** and use it to experiment with a personalized greeting in a classic Quote of the Day UI.

## Prerequisites
* Node.js 18+

> * Azure App Configuration instance with a Front Door endpoint exposing it

Reviewer: ```suggestion
* Azure Front Door endpoint configured with Azure App Configuration as its origin.
```
